backend/substrapp/tasks/utils.py

```diff
@@ -149,3 +150,15 @@ def wrapper(*args, **kwargs):
         except Exception as e:
             logging.exception(e)
     return wrapper
+
+
+def fetch_old_algo_image_names(max_duration):
+
+    logger.info(f'Fetch image names older than {max_duration}')
+
+    return k8s_fetch_old_algo_image_names(max_duration)
+
+
+def garbage_collector():
+    logger.info('Launch garbage collect on docker-registry')
+    k8s_docker_registry_garbage_collector()
```

Comment:
Is there an advantage to having these two methods?

Reply:
What do you mean by that ?

Reply:
These functions are just wrappers: they do nothing but write a log and call another function. Maybe just write the log from the wrapped function and delete the wrapper?

Reply:
Done